packages/vscode-extension/package.json

```diff
@@ -44,6 +44,16 @@
 				"title": "Deploy Project",
 				"category": "SuiteCloud"
 			},
+			{
+				"command": "suitecloud.importfiles",
+				"title": "Import Files",
+				"category": "SuiteCloud"
+			},
+			{
+				"command": "suitecloud.importfile",
+				"title": "Import Single File to Account",
```

Comment:
This seems not right, I think it should be `Import Single File from Account` . Probably this one and plural form should be reviewed by @dforacle .

Reply:
I have fixed but let's wait for David Flett comment.

Reply:
Ok, finally I have put the message from David. I would resolve this conversation then.